cmd/crossplane/core/certs.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package oci
+package core
```

Comment:
What still uses the functionality in this file? I couldn't immediately find anything.

Reply:
It's used here in the same package: https://github.com/crossplane/crossplane/blob/e312b6e92b85d781718f20a29547f69ad6e7558a/cmd/crossplane/core/core.go#L237